onionshare/onion.py

```diff
@@ -316,7 +316,8 @@ def connect(self, settings=False, config=False, tor_status_update_func=None):
             # guessing the socket file name next
             if not found_tor:
                 try:
-                    if self.system == 'Linux':
+                    plat = common.get_platform()
+                    if plat == 'Linux' or plat == 'BSD':
                         socket_file_path = '/run/user/{}/Tor/control.socket'.format(os.geteuid())
```

Comment:
A minor nitpick - nothing wrong here - but rather than initialise a new variable 'plat' here, perhaps it would be better to be consistent and change line 134 of onion.py ```self.system = platform.system()``` to ```self.system == common.get_platform()``` ? 

And then you can just make this ```if self.system == 'Linux' or self.system == 'BSD'```? 

What do you think?